tests/TestCase/View/Widget/DateTimeWidgetTest.php

```diff
@@ -861,14 +862,14 @@ public function testRenderMinuteWidget()
         ], $this->context);
         $this->assertContains('<select name="date[minute]" data-foo="test">', $result);
         $this->assertContains(
-            '<option value="00">00</option>',
+            '<option value="00">0</option>',
             $result,
-            'contains 00'
+            'contains 0'
         );
         $this->assertContains(
-            '<option value="05">05</option>',
+            '<option value="05">5</option>',
```

Comment:
no, for generate numbers for years, months, days use  DateTimeWidget::_generateNumbers().
and Keep in mind that the numbers may be other than Latin.[example picture](https://www.dropbox.com/s/cqjiwtrz3euoqzg/Screenshot%202015-06-25%2005.16.01.png?dl=0)